backend/mora/auth/sso.py

```diff
@@ -33,32 +31,12 @@ def init_app(app):
     """
     app.config.update({
         'SAML_METADATA_URL': settings.SSO_SAML_METADATA_URL,
+        'SAML_IDP_INSECURE': settings.SAML_IDP_INSECURE,
+        'SAML_KEY_FILE': settings.SAML_KEY_FILE,
+        'SAML_CERT_FILE': settings.SAML_CERT_FILE,
+        'SAML_AUTHN_REQUESTS_SIGNED': settings.SAML_AUTHN_REQUESTS_SIGNED,
```

Comment:
`SAML_SIGNED_REQUESTS`?

Reply:
The name I used corresponds to the name of the setting in the pysaml2 config, where this boolean eventually ends up.